stl/src/vector_algorithms.cpp

```diff
@@ -55,6 +55,20 @@ extern "C" {
 __declspec(noalias) void __cdecl __std_swap_ranges_trivially_swappable_noalias(
     void* _First1, void* _Last1, void* _First2) noexcept {
 #if !defined(_M_ARM64EC)
+    constexpr size_t _Mask_64 = ~((static_cast<size_t>(1) << 6) - 1);
+    if (_Byte_length(_First1, _Last1) >= 64 && _bittest(&__isa_enabled, __ISA_AVAILABLE_AVX512)) {
+        const void* _Stop_at = _First1;
+        _Advance_bytes(_Stop_at, _Byte_length(_First1, _Last1) & _Mask_64);
+        do {
+            const __m512i _Left  = _mm512_loadu_si512(static_cast<__m512i*>(_First1));
+            const __m512i _Right = _mm512_loadu_si512(static_cast<__m512i*>(_First2));
+            _mm512_storeu_si512(static_cast<__m256i*>(_First1), _Right);
+            _mm512_storeu_si512(static_cast<__m256i*>(_First2), _Left);

```

Comment:
Wrong typedef `__m256i` used instead of `__m512i`. Fixed in the next bunch of commits.